ethereum/contracts/relayer/coreRelayer/CoreRelayerDelivery.sol

```diff
@@ -420,12 +420,18 @@ abstract contract CoreRelayerDelivery is CoreRelayerBase, IWormholeRelayerDelive
       receiverValueRefundAmount + transactionFeeRefundAmountPostForward;
     
     //Refund the user
-    refundStatus = payRefundToRefundAddress(
+    try this.payRefundToRefundAddress(
```

Comment:
So clean 👌

Reply:
Calling yourself as an external contract is so clutch.